CSS-Fonts (#3044)
Codecademy/docs: content/css/concepts/typography/terms/font-weight/font-weight.md

```css
p {
This code snippet give the text a semi-bold style
  font-weight: 600;
}
```

> **Note:** While using font weight, we should be careful. The way it will be rendered can differ from browser to browser, and for certain languages, it is not advised to use font weight. This includes languages with complex scripts such as Chinese, Japanese, etc.

Reviewer: ```suggestion

> **Note:** Font weight will differ in how it's rendered from browser to browser. For certain languages, it is not advised to use font weight. This includes languages with complex scripts such as Chinese, Japanese, etc.
```

>   font-weight: 600;
> }

Reviewer 2: [Suggestion] This example is missing some context here. Consider viewing my feedback on [this line](https://github.com/Codecademy/docs/pull/3044/files#r1302110496) to help you. 
```suggestion
  font-weight: 600;
}
```
